pkg/collect/mysql.go

```diff
@@ -18,15 +18,37 @@ func Mysql(c *Collector, databaseCollector *troubleshootv1beta2.Database) (Colle
 	if err != nil {
 		databaseConnection.Error = err.Error()
 	} else {
+		defer db.Close()
 		query := `select version()`
 		row := db.QueryRow(query)
+
 		version := ""
 		if err := row.Scan(&version); err != nil {
 			databaseConnection.Error = err.Error()
 		} else {
 			databaseConnection.IsConnected = true
 			databaseConnection.Version = version
 		}
+
+		rows, err := db.Query("SHOW VARIABLES")
+
+		if err != nil {
+			databaseConnection.Error = err.Error()
+		} else {
+			defer rows.Close()
+
+			variables := map[string]string{}
+			for rows.Next() {
+				variable := MySQLVariable{}
```

Comment:
This should just be `var key, value string`

Reply:
Oh nice, thanks for the suggestion.  Just made that update.